misc/metrics/src/swarm.rs

```diff
@@ -20,15 +20,19 @@
 
 use crate::protocol_stack;
 use prometheus_client::encoding::text::Encode;
-use prometheus_client::metrics::counter::Counter;
-use prometheus_client::metrics::family::Family;
+use prometheus_client::metrics::{
+    counter::Counter,
+    family::Family,
+    histogram::{exponential_buckets, Histogram},
+};
 use prometheus_client::registry::Registry;
 
 pub struct Metrics {
     connections_incoming: Family<AddressLabels, Counter>,
     connections_incoming_error: Family<IncomingConnectionErrorLabels, Counter>,
 
     connections_established: Family<ConnectionEstablishedLabels, Counter>,
+    connection_establishment_duration: Family<ConnectionEstablishedLabels, Histogram>,
```

Comment:
```suggestion
    connections_establishment_duration: Family<ConnectionEstablishedLabels, Histogram>,
```

To be consistent with the other metric names.